tasks/src/reinforcement-learning/data.ts

```diff
@@ -0,0 +1,35 @@
+import type { TaskDataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			description: "A subset of widely used Datasets for Data Driven Deep Reinforcement Learning (D4RL)",
+			id:          "edbeeching/decision_transformer_gym_replay",
+		}
+	],
+    demo:     {
+		inputs:  [],
+		outputs: [],
+	},
+    metrics: [{
+        description: "Mean reward per episode obtained after running policy for a certain number of evaluation episodes ",
+        id:          "Mean Reward",
+    }],
+    isPlaceholder: true,
+	models: [
+        {
+			description: "A Decision Transformer model trained on expert trajectories sampled from the Gym Hopper environment",

```

Comment:
it would be nice to simplify this line for a broader audience (e.g. software developers) 🙂